bundler.d/test.rb

```diff
@@ -6,6 +6,7 @@
   gem 'ci_reporter', '>= 1.6.3', "< 2.0.0", :require => false
   gem 'rdoc'
   gem 'minitest', '~> 4.7', :platforms => :ruby_19
```

Comment:
And I suppose on Ruby 2.0 and 2.1, we're actually using minitest from Ruby core, with its test-unit compatibility.  Should it just be test-unit throughout?